lib/class-wp-duotone-gutenberg.php

```diff
@@ -74,25 +74,60 @@ class WP_Duotone_Gutenberg {
 
 	/**
 	 * An array of Duotone SVG and CSS output needed for the frontend duotone rendering based on what is
-	 * being output on the page. Organized by a slug of the preset/color group and the information needed
+	 * being output on the page. Organized by an id of the preset/color group and the information needed
 	 * to generate the SVG and CSS at render.
 	 *
 	 * Example:
 	 *  [
-	 *      'blue-orange' => [
+	 *      'wp-duotone-blue-orange' => [
 	 *          'slug'  => 'blue-orange',
 	 *          'colors' => [ '#0000ff', '#ffcc00' ],
 	 *      ],
 	 *      'wp-duotone-000000-ffffff-2' => [
-	 *          'slug' => 'wp-duotone-000000-ffffff-2',
+	 *          'slug' => '000000-ffffff-2',
 	 *          'colors' => [ '#000000', '#ffffff' ],
 	 *      ],
 	 * ]
```

Comment:
This changed from being keyed by `slug` to being keyed by `id` so that the `id` didn't have to get recomputed again later. The key in the second part of the example (`wp-duotone-000000-ffffff-2`) was wrong before, but it's right now which is why it didn't change.